app.js

```diff
@@ -52,7 +53,10 @@ app.locals.franceConnectKitUrl = `${config.FC_URL}${config.FRANCE_CONNECT_KIT_PA
 
 app.get('/', (req, res) => res.render('pages/home'));
 
-app.get('/login', (req, res) => res.status(200).render('pages/login', {}));
+app.get('/login', (req, res) => {
+  const scopesDefined = `${config.MANDATORY_SCOPES} ${config.FC_SCOPES}`.split(' ');
```

Comment:
« scopesDefined » seems to refer to the list of all defined scopes though it is used as the list of preselected scopes. 

You should name this : « scopesSelectedByDefault »